### PR TITLE
refactor: crawler/cloud resource manager

### LINKED ISSUE #213: Subtasks: Implement crawl factory for Cloudresourcemanager resource.
#153

### PR DESCRIPTION
**Pull request by peb-peb**
## NOTE

closes #213 
ref to #153 

@mshudrak @ZetaTwo

### CODE REVIEW

**Comment by peb-peb** [comment]
Conflicts Resolved!

File: src/gcp_scanner/crawler/cloud_resource_manager_project_info_crawler.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import logging
+import sys
+from typing import Dict, Any
+
+from googleapiclient import discovery
+
+from gcp_scanner.crawler.interface_crawler import ICrawler
+
+
+class CloudResourceManagerProjectInfoCrawler(ICrawler):
```
Comment (mshudrak) [inline src/gcp_scanner/crawler/cloud_resource_manager_project_info_crawler.py:24]: Is there any specific reason to have CloudResourceManager in the name? I'd just omit it and have ProjectInfoCrawler here and everywhere else

**Review by mshudrak — CHANGES_REQUESTED** [review]
Looks good, I have one comment/suggestion. PTAL.

File: src/gcp_scanner/crawler/cloud_resource_manager_project_info_crawler.py (hunk 1 above)
Comment (peb-peb) [reply src/gcp_scanner/crawler/cloud_resource_manager_project_info_crawler.py:24]: I was going with the pattern found in #153, like:

image
image

Both of these have `compute` or `dns` in the prefix of their Class names. Maybe we can also go with `CRM` or put all these in a separate sub-folder to group them without explicitly mentioning it.

Comment (mshudrak) [reply src/gcp_scanner/crawler/cloud_resource_manager_project_info_crawler.py:24]: I see. It is ok, let's keep it. CRM will be confusing

**Review by mshudrak — APPROVED** [review]
LGTM

**Pull request merged**